cypress/e2e/organization.spec.js

```diff
@@ -75,7 +76,7 @@ describe('Organization JSON-LD', () => {
             addressCountry: 'France',
           },
         ],
-        contactPoints: [
```

Comment:
Lets leave this as `contactPoints` so that the deprecated prop of `contactPoints` is still tested.